### ISSUE TITLE
Custom sort function

### ISSUE THREAD

**Issue by stefangabos** [crlcu/multiselect#129]
It's not clear from the documentation how to use a callback function for the `sort` parameter.

Basically, I want the panel on the left to be sorted alphabetically while the one on the right to have `keepRenderingSort`. The callback function attached to `sort` gets a single parameter, the `option` itself (it would be great to also get the panel where it comes from) but returning `true` or `false` seem to have the same effect of not making the panels work as if they would have `keepRenderingSort` set to `true`. 

So, in short, I want to prevent the sorting but only for the right panel. Is this possible?

Thanks!

**Comment by crlcu** [comment]
Hi @stefangabos,

Nope that's not possible, but I don't mind adding this feature, or maybe you would like to do it and create a pull request?

**Comment by dev22m** [comment]
Hi crlcu,
I am using your multiselect plugin and its look great.
But when i have implemented V2.4.0 after that when i m select item from left and move to right that time right side items not getting automatically selected. in V2.0 its automatically selected all items which we have moved from left to right.

So can you please help me in this.
